pawno/include/PPC_PlayerCommands.inc

```diff
@@ -1,27 +1,28 @@
 // This is just a joke command
-COMMAND:me(playerid,params[])
+COMMAND:me(playerid, params[])
```

Comment:
Please undo all changes to this lines as they are mainly a matter of personal preferences, e.g. leaving blank spaces or removing the quotes. Apart from that changes to the color of messages are always welcome. We need to make more use of the color definitions. That would require an own pull request.

Reply:
Yea, I was thinking about this but i found different things:

`COMMAND:pm(playerid, params[])`

`COMMAND:createhouse(playerid, params[])`

`COMMAND:delhouse(playerid, params[])`
and...

As you can see there's one SPACE between them 
Only "me" and "givelicense" have no SPACE

Reply:
Oh I see. Could you please add the blank space for `/givelicense` as well?